raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -43,26 +43,22 @@ library NettingChannelLibrary {
 
 
     modifier notSettledButClosed(Data storage self) {
-        if (self.settled > 0 || self.closed == 0)
-            throw;
+        assert(self.settled <= 0 && self.closed > 0);
```

Comment:
Should be a `require`. All these are states of the contract which can be enountered, not invalid states that can never be reached.